credentials.go

```diff
@@ -0,0 +1,17 @@
+package grafton
+
+import (
+	"regexp"
+)
+
+// NameRegexp is the regular expression for determining if a credential name is
+// valid or not.
+//
+// Specified in Shell and Utilities volume of IEEE 1003.1-2001.
+var NameRegexp = regexp.MustCompile("^[A-Z][A-Z0-9_]{0,127}$")
```

Comment:
you could export the uncompiled string form of the regexp as a const, and then use it in acceptance/credentials.go's output msg.